Automattic/wp-calypso
# Stats: Update to a higher resolution Woo icon.

#### Proposed Changes

Adds a higher resolution Woo icon and controls the display size via CSS. This fixes the fuzzy icon on HiDPI screens.

#### Testing Instructions



* Check out this branch.
* Run `yarn workspace @automattic/components run storybook` to fire up Storybook.
* View the Mobile Promo Card → Default to see the Jetpack card.
* Toggle to the Woo card and confirm the layout doesn't shift around.
* Confirm the Woo icon doesn't look fuzzy on a HiDPI/Retina display.

#### Pre-merge Checklist





Related to #70462.
